src/main/java/com/revature/models/Wishlist_products.java

```diff
@@ -0,0 +1,24 @@
+package com.revature.models;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
+public class Wishlist_products {
```

Comment:
Not a fan of underscores in class names. Rename class to `WishlistProducts` and add `@Table(name = "wishlist_products")` annotation.